src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -671,4 +673,15 @@ public TeamDetailsBundle getTeamDetailsForStudent(StudentAttributes student) {
         return null;
     }
 
+    private String addDuplicateEmailInfo(String sanitizedLine, String[] linesArray, int isDuplicate) {
```

Comment:
1. Write header comments.
1. Try changing the method header to take in `String duplicateLine, String sanitizedLine` instead i.e. remove `String[] linesArray, int isDuplicate`. See if it looks better. We don't need to pass in the entire array if we are only using 1 element of the array.
1. Rename the method name: the caller is doing `invalidityInfo.add(addDuplicateEmailInfo(...));` which looks weird. Renaming it to `duplicateEmailInfo` sounds better.
1. Rename `sanitizedLine`.